test/test.googleauth.ts

```diff
@@ -408,47 +435,51 @@ describe('GoogleAuth', () => {
     it('should read the stream and create a jwt', (done) => {
       // Read the contents of the file into a json object.
       const fileContents =
-          fs.readFileSync('./ts/test/fixtures/private.json', 'utf-8');
+          fs.readFileSync('./test/fixtures/private.json', 'utf-8');
       const json = JSON.parse(fileContents);
 
       // Now open a stream on the same file.
-      const stream = fs.createReadStream('./ts/test/fixtures/private.json');
+      const stream = fs.createReadStream('./test/fixtures/private.json');
 
       // And pass it into the fromStream method.
       const auth = new GoogleAuth();
       auth.fromStream(stream, (err, result) => {
         assert.equal(null, err);
-
-        // Ensure that the correct bits were pulled from the stream.
-        assert.equal(json.private_key, result.key);
-        assert.equal(json.client_email, result.email);
-        assert.equal(null, result.keyFile);
-        assert.equal(null, result.subject);
-        assert.equal(null, result.scope);
-
+        assert(result);
+        const jwt = result as JWT;
+        if (result) {
```

Comment:
nit: don't intermingle the two aliases jwt and result. Use jwt here onwards. Or move the cast to inside the if.